.eslintignore

```diff
@@ -0,0 +1,2 @@
+lib/xregexp.js
+spec/*
```

Comment:
Sorry for being nit picky but please add a final newline at the end of files. It's an Atom default, [it's how UNIX tools expect files](https://stackoverflow.com/a/729795/5587737) and having to add it later just adds diff noise.